src/lib.rs

```diff
@@ -1,21 +1,23 @@
 //! # Chrono: Date and Time for Rust
 //!
 
-//! Chrono aims to provide all functionality needed to do correct operations on dates and times in the
-//! [proleptic Gregorian calendar](https://en.wikipedia.org/wiki/Proleptic_Gregorian_calendar):
+//! Chrono aims to provide all functionality needed to do correct operations on dates and times in
```

Comment:
How did you reflow? It doesn't look very consistent.

Reply:
Manually against a line at the 100-characters column in my editor. And sorry, it should be a bit better now. I missed a few lines, and later changes to a few links messed up parts that I did before.